src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -213,7 +213,7 @@ form.comment {
 
 .code-line {
     white-space: pre-wrap;
-    font-family: monospace;
+    font-family: Consolas, monospace;
```

Comment:
what is this change for?

Reply:
So firefox uses Consolas by default. Chrome was using it already.

Reply:
The default monospace font on some browser+OS combinations isn't particularly pleasant (e.g. FF+Windows is Courier New 😥). This helps for Windows users. (Adding Menlo would help Mac users).

Reply:
Wikipedia says:
>It is not installed as a user-accessible typeface by default, although it is included with and used by Xcode.